app/models/concerns/account_settings.rb

```diff
@@ -110,8 +110,10 @@ def solr_collection_options
   end
   # rubocop:enable Metrics/BlockLength
 
-  def public_settings
-    all_settings.reject { |k, v| Account.private_settings.include?(k.to_s) || v[:disabled] }
+  def public_settings(is_superadmin: false)
+    settings = all_settings
+    settings = superadmin_settings(is_superadmin) unless is_superadmin
```

Comment:
```suggestion
    settings = superadmin_settings unless is_superadmin
```

Sorry I hadn't refreshed this page and see you applied similar changes already. I don't think you will need to pass is_superadmin to superadmin_settings though. Rubocop is complaining that it's an unused variable.